css/pizzas.css

```diff
@@ -0,0 +1,15 @@
+
+#annualBarChartHolder {
+	width: 100%;
+	height: 600px;
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+	margin-right: 100px;
+}
+
+#pizzaCharts {
+	width: 100%;
+	height: 100%;
+	display: flex;
```

Comment:
I think we probably want to have a `@media` query somewhere here in this file, so that on mobile devices we can change the `flex-direction` to `column`, so that the pie chart shows below the bar chart on mobile, instead of disappearing off the edge of the screen.